src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -378,8 +379,12 @@ public function denormalize(mixed $data, string $type, string $format = null, ar
             }
         }
 
-        if ($extraAttributes) {
-            throw new ExtraAttributesException($extraAttributes);
+        if (!empty($extraAttributes)) {
```

Comment:
```suggestion
        if ($extraAttributes) {
```
You can revert that line, as we try to use `empty` as less as possible

Reply:
Wasn't aware of this. Just read some articles about it. Will fix